docs/proposals/registry/sample-vs-stack.md

```diff
@@ -0,0 +1,195 @@
+      │   │── deploy
+      │   │   └── deployment-manifest.yaml
+      │   └── devfile.yaml
+                …
+
+
+With the introduction of the samples in the registry, the samples will be stored under a similar directory structure if the sample source is stored under the registry source repo:
+
+    ── stacks
+      │── java-maven
+      │   └── devfile.yaml
+      │
+      │── java-openliberty
+      │   └── devfile.yaml
+      │
+      │── java-quarkus
+      │   └── devfile.yaml
+      │
+      │── nodejs
+      │   │── build
+      │   │   └── Dockerfile
+      │   │── deploy
+      │   │   └── deployment-manifest.yaml
+      │   └── devfile.yaml
+                …
+    ── samples
+      │── nodejs-sample
+      │   │── .gitignore
+      │   │       LICENSE
+      │   │       README.md
+      │   │       devfile.yaml
+      │   └── src
+      │              └── Dockerfile
+      │                       package-lock.json
+      │                       package.json
+      │                       server.js
+      │
+      │── springboot-sample
+      │   │── devfile.yaml
+      │   │       .gitignore
+      │   │       LICENSE
+      │   │       README.md
+      │   │       mvnw
+      │   │       mvnw.cmd
+      │   │       pom.xml
+      │   └── src
+      │              │── main
+      │              │      java
+      │              │           │── com
+          │              │             ...
+                …
+
+
+## Registry query results
+The information on the registry index (index.json) is different between a stack and a sample. The differences are based on how the two will be used.
+
+    [
+        {
+            "name": "java-maven",
+            "version": "1.1.0",
+            "displayName": "Maven Java",
+            "description": "Upstream Maven and OpenJDK 11",
+            "type": "stack",
+            "tags": [
+                "Java",
+                "Maven"
+            ],
+            "projectType": "maven",
+            "language": "java",
+            "links": {
+                "self": "devfile-catalog/java-maven:latest"
+            },
+            "resources": [
+                "devfile.yaml"
+            ],
+            "starterProjects": [
+                "springbootproject"
+            ]
+        },
+        {
+            "name": "nodejs-basic",
+            "displayName": "Basic NodeJS",
+            "description": "A simple Hello world NodeJS application",
+            "icon": "nodejsIcon.svg",
+            "type": "sample",
+            "tags": [
+                "NodeJS",
+                "Express"
+            ],
+            "projectType": "nodejs",
+            "language": "nodejs",
+            "git": {
+                "remotes": {
+                    "origin": "https://github.com/redhat-developer/devfile-sample/"
+                }
+            }
+        }
+    ]
+
+The bold entries highlighted the differences between a stack entry vs a sample entry in the devfile index registry. 
```

Comment:
The `bold entries` are not displayed after moving docs from google docs, we need manually added them again by using markdown.